test/storage/metric_test.cpp

```diff
@@ -0,0 +1,515 @@
+#include <memory>
+#include <random>
+#include <string>
+#include <thread>  //NOLINT
+#include <unordered_map>
+#include <vector>
+#include "storage/garbage_collector.h"
+#include "storage/metric/database_metric.h"
+#include "storage/metric/stats_aggregator.h"
+#include "storage/metric/thread_level_stats_collector.h"
+#include "storage/metric/transaction_metric.h"
+#include "transaction/transaction_defs.h"
+#include "transaction/transaction_manager.h"
+#include "util/metric_test_util.h"
+#include "util/test_harness.h"
+#include "util/transaction_test_util.h"
+
+namespace terrier {
+
+namespace settings {
+class SettingsManager;
+}
```

Comment:
I think you can remove this namespace declaration from here.

Reply:
Settings manager is not yet merged, so there is not a def file we can include.